vue-frontend/src/stores/index.js

```diff
@@ -0,0 +1,47 @@
+import { defineStore } from "pinia";
```

Comment:
We can name files module wise like `jobs.js`

Reply:
Yeah 👍